__tests__/auth.test.js

```diff
@@ -0,0 +1,13 @@
+import { loginDeveloper } from './shared/commands';
+
+describe('login', () => {
+  it('should hit the login endpoint and return a message', async () => {
+    const data = {
+      email: 'test@email.com',
+      password: 'password',
+    };
+    const login = await loginDeveloper(data);
```

Comment:
how does this test work? wouldn't you need to create a new developer account in the test with this email and password in order for it to exist in the database?

Reply:
Done